Iceberg-TipUI/IceTipEditProjectDialogPresenter.class.st

```diff
@@ -246,9 +246,8 @@ IceTipEditProjectDialogPresenter >> initializeDirectoryTree [
 						  evaluated: #basename;
 						  yourself));
 		roots: { self model fileSystem };
-		children: [ :each | each directories ];
-		whenSelectionChangedDo: [ :announcement | 
-			self sourceDirectorySelectionChanged: announcement selectedPaths ]
+		children: [ :each | each directories reject: [ :dir | dir basename endsWith: '.git' ] ];
```

Comment:
```suggestion
		children: [ :each | each directories reject: [ :dir | dir basename = '.git' ] ];
```

Reply:
This should not happen much but we could have a file or directory ending with .git that is not the .git folder.